src/tools.js

```diff
@@ -60,6 +60,19 @@ export function isValidDate(date) {
   return !isNaN(date.getTime())
 }
 
+export function omitted(propNames) {
+  /*
+  Returns a function that returns all props without the ones whose name is in `propNames.
```

Comment:
Doc should also be added to the `README.md`.

Reply:
on another pr